frontend/express/public/javascripts/countly/countly.common.js

```diff
@@ -228,6 +228,9 @@
         * @returns {string} encode string
         */
         countlyCommon.encodeSomeHtml = function(html, options) {
+            if (countlyGlobal.company) {
+                html.replace("Countly", countlyGlobal.company);
+            }
```

Comment:
I think ticket meant, not applying replacement in this method, but rather applying encodeSomeHtml to each and every localization string